arch.sh

```diff
@@ -17,34 +17,34 @@ setup_network_config() {
     CONFIGFILE="$FOLD/hdd/etc/systemd/network/50-hetzner.network"
     UDEVFILE="$FOLD/hdd/etc/udev/rules.d/80-net-setup-link.rules"
 
-    echo -e "### ${COMPANY} - installimage" > "$UDEVFILE"
-    echo -e "# device: $1" >> "$UDEVFILE"
+    echo "### ${COMPANY} - installimage" > "$UDEVFILE"
+    echo "# device: $1" >> "$UDEVFILE"
     printf 'SUBSYSTEM=="net", ACTION=="add", DRIVERS=="?*", ATTR{address}=="%s", ATTR{dev_id}=="0x0", ATTR{type}=="1", KERNEL=="eth*", NAME="%s"\n' "$2" "$1" >> "$UDEVFILE"
 
-    echo -e "### $COMPANY - installimage" > "$CONFIGFILE"
-    echo -e "# device: $1" >> "$CONFIGFILE"
-    echo -e "[Match]" >> "$CONFIGFILE"
-    echo -e "MACAddress=$2" >> "$CONFIGFILE"
-    echo -e "" >> "$CONFIGFILE"
+    { echo "### $COMPANY - installimage"
+    echo "# device: $1"
+    echo "[Match]"
+    echo "MACAddress=$2"
+    echo ""; } > "$CONFIGFILE"
```

Comment:
In centos.sh there's a newline instead of the semicolon, I'd prefer that everywhere